content/index.yaml

```diff
@@ -2,6 +2,7 @@
 meta:
   title: 2022 CC Community Music Awards
   heading: 2022 Nominees & Winners
+  description: Annual CC Community Music Awards
   description: The fifth annual CC Community Music Awards
```

Comment:
Looks like we've now got two descriptions. We should only have one.

Reply:
Ah, yes I saw that, but didn't notice the lack of the minus. I was trying to genericize the index.html, which I think is a worthy long-term goal. That way, the new year will just be adding the year.yml file. I'll fix

Reply:
I'm going to resolve this as when I go to Commits the new commit seems to be there. I guess we shall see